app/controllers/FileChecksController.scala

```diff
@@ -0,0 +1,43 @@
+package controllers
+
+import java.util.UUID
+
+import auth.TokenSecurity
+import configuration.{GraphQLConfiguration, KeycloakConfiguration}
+import javax.inject.{Inject, Singleton}
+import org.pac4j.play.scala.SecurityComponents
+import play.api.i18n.I18nSupport
+import play.api.mvc.{Action, AnyContent, Request, RequestHeader}
+import services.ConsignmentService
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class FileChecksController @Inject()(val controllerComponents: SecurityComponents,
+                                     val graphqlConfiguration: GraphQLConfiguration,
+                                     val keycloakConfiguration: KeycloakConfiguration,
+                                     consignmentService: ConsignmentService
+                                    )(implicit val ec: ExecutionContext) extends TokenSecurity with I18nSupport {
+
+  private def getRecordProcessingProgress(request: Request[AnyContent], consignmentId: UUID)
+                                         (implicit requestHeader: RequestHeader): Future[(Int, Int)] = {
```

Comment:
A small case class would be nice here so that the return value doesn't have to be a tuple, so that you can give the values names and don't have to get the order of the Ints correct.

Reply:
I agree, i'll change it :+1: